lte/gateway/c/core/oai/tasks/nas/nas_proc.c

```diff
@@ -892,8 +898,9 @@ status_code_e nas_proc_pdn_disconnect_rsp(
   int rc            = RETURNerror;
   emm_sap_t emm_sap = {0};
 
-  OAILOG_DEBUG(
-      LOG_NAS_EMM, "Received pdn_disconnect_rsp for ue id %u\n",
+  OAILOG_DEBUG_UE(
+      LOG_NAS_EMM, emm_sap.u.emm_esm.ctx->_imsi64,
```

Comment:
This will always be invalid (since `emm_sap` its being allocated here), let's not change this one

Reply:
Resolved